src/main/java/Baymax.java

```diff
@@ -0,0 +1,92 @@
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Baymax {
+    public static void main(String[] args) {
+        String name = "Baymax";
+        System.out.println("Hello! I'm " + name + ", your personal task manager." + System.lineSeparator()
+                + "What can I do for you today?" + System.lineSeparator() + System.lineSeparator());
+
+        ArrayList<Task> taskArrayList = new ArrayList<>();
```

Comment:
plural form for collection variable name would be more appropriate such as "tasks"